build.cmd

```diff
@@ -113,10 +140,8 @@ call :build %__args%
 goto :AfterBuild
 
 :build
-:: *** start WCF Content <WCF uses additional msbuild args> ***
-%_buildprefix% msbuild "%_buildproj%" %_defaultBuildConfig% /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=diag;LogFile="%_buildlog%";Append "/l:BinClashLogger,%_binclashLoggerDll%;LogFile=%_binclashlog%" %__args% %_buildpostfix%
-set BUILDERRORLEVEL=!ERRORLEVEL!
-:: *** end WCF Content ***
+%_buildprefix% msbuild "%_buildproj%" /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=normal;LogFile="%_buildlog%";Append "/l:BinClashLogger,%_binclashLoggerDll%;LogFile=%_binclashlog%" !unprocessedBuildArgs! %_buildpostfix% %OfficialBuildIdArg%
+set BUILDERRORLEVEL=%ERRORLEVEL%
```

Comment:
The corefx script was updated in PR dotnet/corefx #7724 with `setlocal EnableDelayedExpansion` in order to support new logic being added.

You are correct, I should have left the '!'s around ERRORLEVEL.